packages/ProgressAccordion/src/ProgressAccordion.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import PropTypes from "prop-types";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import { visuallyHidden } from "@paprika/stylers/lib/includes";
+import Item from "./components/Item";
+import Indicator from "./components/Indicator";
+import Responses from "./components/Responses";
+import {
+  accordionStyles,
+  itemStyles,
+  activeItemStyles,
+  activeLabelStyles,
+  activeStatusStyles,
+} from "./ProgressAccordion.styles";
+
+const propTypes = {
+  a11yText: PropTypes.string,
+  activeIndex: PropTypes.number,
+  activeStatus: PropTypes.node,
+  children: PropTypes.node,
+};
+
+const defaultProps = {
+  a11yText: null,
+  activeIndex: 0,
+  activeStatus: null,
+  children: null,
+};
+
+function filterChildren(children) {
+  return React.Children.toArray(children).filter(child => child.type && child.type.displayName === Item.displayName);
+}
+
+const ProgressAccordion = props => {
+  const { a11yText, activeIndex, activeStatus, children, ...moreProps } = props;
+
+  const I18n = useI18n();
+
+  const validChildren = filterChildren(children);
+
+  const getLabel = (label, index) => {
+    return activeIndex === index ? (
+      <div css={activeItemStyles}>
+        <div css={activeLabelStyles}>
+          {label}
+          <span css={visuallyHidden}>. {I18n.t("progressAccordion.active")}</span>
+        </div>
+        <div css={activeStatusStyles}>{activeStatus}</div>
+      </div>
+    ) : (
+      label
+    );
+  };
+
+  return (
+    <div css={accordionStyles} {...moreProps} role="list" aria-label={a11yText}>
+      {validChildren.length > 0 &&
+        validChildren.map((child, index) => {
+          const isComplete = activeIndex > index;
+          const indicatorProps = {
+            isComplete,
+            isActive: activeIndex === index,
+            isLast: index === validChildren.length - 1,
+          };
+
+          return (
+            // eslint-disable-next-line react/no-array-index-key
+            <div css={itemStyles} key={index} role="listitem">
```

Comment:
@nahumzs said it's okay to break the rules! 😁 
This component is unlikely to re-render these items, and even if it did occasionally, the impact on performance is not a significant concern. Simplifying the API for the consumer is preferred.

Reply:
It's not so much about performance as it is about correctness though? https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318
Assuming we're following the robustness principle, we should be able to re-render the component with a different set of items without breaking 🤷‍♂

Reply:
> It is true that sometimes generating new ids is redundant and may be avoided. To help you decide, I put together three conditions which these examples have in common:

> - the list and items are static–they are not computed and do not change;
> - the items in the list have no ids;
> - the list is never reordered or filtered.

> When all of them are met, **you may safely use the index as a key**.

So I think since we would never push an item into the list, remove one from the middle, or re-order the list it should be safe?  In fact, the list won't ever change _at all_ in either of the use cases we have lined up for this component.

Reply:
Ok, as long as we understand why it is safe to do.
I can still imagine cases where this can be necessary (e.g., you pick something in step 1 that makes step 3 irrelevant and we want to remove it). But let's invoke YAGNI for now.

While not directly related, this reminds me of how people sometimes assume that a certain prop will never change its value during a component's lifecycle, when in reality the component model implies you should be able to accept a new prop value after the initial mount (e.g. https://overreacted.io/writing-resilient-components/#dont-stop-the-data-flow-in-rendering)